### PR TITLE
add full journey route

### PR DESCRIPTION
**Pull request by gokhakan**

### CODE REVIEW

File: src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import org.junit.Assert;
+
+public class FullJourneyRouteSteps {
+
+    // this step needs to be replaced with Given I am on Orchestrator stub in CommonSmokeSteps
+    @Given("I am on Orchestrator Stub URL")
+    public void i_am_on_orchestrator_stub_url() {
+        Driver.get().get(ConfigurationReader.getOrchestratorUrl());
+        BrowserUtils.waitForPageToLoad(10);
+    }
```
Comment (Wynndow) [inline src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java:19]: I think you can drop this now in favour of the step defined in CommonSmokeSteps

File: src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
+package gov.di_ipv_core.step_definitions;
+
+import gov.di_ipv_core.pages.*;
```
Comment (Wynndow) [inline src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java:3]: We try to not use wildcard imports like this. Intellij uses the "*" automatically if we import more than a small handful of things from the same package. You can adjust intellij to not do this:

Preferences -> Editor -> Code Style -> Java -> Imports
Make sure that the `Use single class import` option is enabled.
Set `Class count to use import with '*'` and `Names count to use static import with '*'` to 999

File: src/test/resources/features/fullJourneyRoute.feature [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  Scenario: UI and Accessibility tests of `Full journey route`
+    Given I am on Orchestrator Stub URL
+    When I click on `Full journey route`
+    Then I should be on `You’ve signed in to your GOV.UK account` page
+    And the number of severe and serious issues detected must be zero
+    When I click `Continue`
+    Then I should be on `Enter your details exactly as they appear on your UK passport` page
+    When I enter Mary Watson's details and click Continue
+#    Then I should be on UK Passport (Stub)
+#    When I supply data in JSON format, GPG Strength 4, Validity 1 and click `Submit data and generate auth code`
```
Comment (Wynndow) [inline src/test/resources/features/fullJourneyRoute.feature:13]: Can these commented lines be removed?

Comment (gokhakan) [reply src/test/resources/features/fullJourneyRoute.feature:13]: yes. they are now redundant.

File: src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java (hunk 1 above)
Comment (gokhakan) [reply src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java:19]: Yes. This is a duplication.

File: src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java (hunk 2 above)
Comment (gokhakan) [reply src/test/java/gov/di_ipv_core/step_definitions/FullJourneyRouteSteps.java:3]: Ok.

**Review by Wynndow — APPROVED** [review]

**Pull request merged**